tools/dep_updaters/update-ada.sh

```diff
@@ -35,13 +38,14 @@ cleanup () {
 trap cleanup INT TERM EXIT
 
 ADA_REF="v$NEW_VERSION"
-ADA_ZIP="ada-$NEW_VERSION.zip"
+ADA_ZIP="ada-$ADA_REF.zip"
```

Comment:
why this change? It doesnt seem related to the PR

Reply:
For consistency with the other deps that logs the checksum in the format `SHA256 (packagename-v1.0.0.zip) = abc123` for e.g. `SHA256 (ada-v2.4.2.zip) = 1c1d96f27307b6f7bd21af87d10d528207f44e904f77a550837037e9def06607`